### DISCUSSION TITLE
3.8的无法多开，能否更新一下

### DISCUSSION THREAD

**Discussion by jekey2000** [huiyadanli/RevokeMsgPatcher#460] [Q&A]
如题，今天刚更新了微信3.8.0.31，结果无法多开，请版主更新一下。

**Comment by China74ya** [comment]
我也是哎

**Comment by YR613** [comment]
是的，是这样的，没法多开了

**Comment by 2776447405** [comment]
重新打一下补丁就好了，更新把文件更新了，需要重新打补丁

**Comment by YR613** [comment]
&nbsp;已经处理，辛苦了，谢谢您


&nbsp;


yryp
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年11月17日(星期四) 晚上7:38
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [huiyadanli/RevokeMsgPatcher] 3.8的无法多开，能否更新一下 (Discussion #460)





 
重新打一下补丁就好了，更新把文件更新了，需要重新打补丁
